sql/src/main/java/org/apache/druid/sql/calcite/planner/CalcitePlannerModule.java

```diff
@@ -40,6 +40,7 @@ public void configure(Binder binder)
     // We're actually binding the class to the config prefix, not the other way around.
     JsonConfigProvider.bind(binder, "druid.sql.planner", PlannerConfig.class);
     JsonConfigProvider.bind(binder, "druid.sql.planner", SegmentMetadataCacheConfig.class);
+    JsonConfigProvider.bind(binder, "druid.sql.planner.operatorConversion", PlannerOperatorConversionConfig.class);
```

Comment:
It is very useful to put the path in the config object:

```java
class PlannerOperatorConversionConfig {
  public static final String CONFIG_PATH = "druid.sql.planner.operatorConversion";
```

And:

```java
JsonConfigProvider.bind(binder, PlannerOperatorConversionConfig.CONFIG_PATH, PlannerOperatorConversionConfig.class);
```

Reply:
good suggestion, fixed